corehq/apps/hqadmin/static/hqadmin/js/project_map.js

```diff
@@ -254,10 +187,24 @@ var projectMapInit = function(mapboxAccessToken) {
             click: function(e) {
                 selectionModel.selectedCountry(feature.properties.name);
                 modalController.showProjectsTable(selectionModel.selectedCountry());
-
+                var country = (feature.properties.name).toUpperCase();
+                $.ajax({
+                    url: "/hq/admin/top_five_projects_by_country/?country=" + country,
+                    datatype: "json",
+                }).done(function(data){
+                    selectionModel.topFiveProjects.removeAll();
```

Comment:
It may be preferable to move this line out of the ajax call. I imagine that if HQ is slow to respond, a user might click on a country, then see the top 5 projects from the last clicked country for a moment before the ajax call finished.